src/fromager/urlutils.py

```diff
@@ -0,0 +1,10 @@
+import os
```

Comment:
I see `threading_utils.py`, maybe follow that pattern and rename to `url_utils.py` ?

Reply:
This is a three line function. Just add it to `resolver`.